packages/mui-lab/src/useAutocomplete/index.d.ts

```diff
@@ -1 +1 @@
-export { useAutocomplete as default } from '@mui/base';
+export { default } from '@mui/material/useAutocomplete';
```

Comment:
The hook was already copied in @mui/matial when we were removing the @mui/base dependency from there. Here, we are just changing the location of where we re-export.